documentation/cluster-controller.adoc

```diff
@@ -302,5 +302,45 @@ The controller itself can be configured through the following environment variab
 
 * `STRIMZI_CONFIGMAP_LABELS`: the Kubernetes/OpenShift label selector used to identify ConfigMaps to be managed by the controller.
 Default: `strimzi.io/kind=cluster`.  
-* `STRIMZI_FULL_RECONCILIATION_INTERVAL` : the interval between periodic reconciliations.
+* `STRIMZI_FULL_RECONCILIATION_INTERVAL`: the interval between periodic reconciliations.
 Default: 120000 ms
+* `STRIMZI_NAMESPACE`: A comma-separated list of namespaces that the controller
+should operate in.
+By default, the only namespace watched is the one in which the controller is deployed.
+See <<multi-namespace, below>> for further details.
+
+[[multi-namespace]]
+==== Watching multiple namespaces
+
+The `STRIMZI_NAMESPACE` environment variable can be used to configure a single controller instance
+to operate in multiple namespaces. For each namespace given, the controller will watch for cluster ConfigMaps
+and perform periodic reconciliation. To be able to do this, the controller's service account needs
+access to the necessary resources in those other namespaces. This can be done by creating a `RoleBinding` in
```

Comment:
This looks kind of "out of place" because we actually do not explain that there is some service account, some roles etc. And now you suddenly jump out with some binding. I think we need to do the whole story about how it is deployed and then this will fit in.